hvac/api/secrets_engines/kv_v2.py

```diff
@@ -59,6 +59,12 @@ def read_configuration(self, mount_point=DEFAULT_MOUNT_POINT):
             mount_point=mount_point,
         )
         return self._adapter.get(url=api_path)
+    
+    def read_secret(self, path, mount_point=DEFAULT_MOUNT_POINT):
+        api_path = utils.format_url('/v1/{mount_point}/data/{path}', mount_point=mount_point, path=path)
+        return self._adapter.get(
+            url=api_path,
+        )
```

Comment:
You should not duplicate the code. It can be easily replaced with
```suggestion
    def read_secret(self, path, mount_point=DEFAULT_MOUNT_POINT):
        return self.read_secret_version(path, mount_point=mount_point)
```